### PR TITLE
Document certified Helm Charts and consolidate Helm docs

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: enterprise-`4.9`
- **JIRA issues**: [RHDEVDOCS-2925](https://issues.redhat.com/browse/RHDEVDOCS-2925)
- **Preview pages**: [Understanding Helm](https://deploy-preview-35759--osdocs.netlify.app/openshift-enterprise/latest/applications/working_with_helm_charts/understanding-helm.html), [Red Hat Certification of Helm charts for OpenShift](https://deploy-preview-35759--osdocs.netlify.app/openshift-enterprise/latest/applications/working_with_helm_charts/understanding-helm.html#red-hat-certification-of-helm-charts-for-openshift), [Installing Helm](https://deploy-preview-35759--osdocs.netlify.app/openshift-enterprise/latest/applications/working_with_helm_charts/installing-helm.html), [Configuring custom Helm chart repositories](https://deploy-preview-35759--osdocs.netlify.app/openshift-enterprise/latest/applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.html), [Working with Helm releases](https://deploy-preview-35759--osdocs.netlify.app/openshift-enterprise/latest/applications/working_with_helm_charts/odc-working-with-helm-releases.html)
- **SME Review**: Completed by @zonggen, @dperaza4dustbit
- **QE review**: Completed by @pmacik 
- **Peer-review**: Completed by @Preeticp

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
@baijum @zonggen @pmacik @Preeticp Please review and provide your feedback. Thanks!

File: applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
+[id="configuring-custom-helm-chart-repositories"]
+
+= Configuring custom Helm chart repositories
+include::modules/common-attributes.adoc[]
+:context: configuring-custom-helm-chart-repositories
+
+toc::[]
+
+You can install Helm charts on an {product-title} cluster using the CLI. The *Developer Catalog*, in the *Developer* perspective of the web console, displays the Helm charts available in the cluster. By default, it lists the Helm charts from the Red Hat Helm chart repository. For a list of the charts see link:https://redhat-developer.github.io/redhat-helm-charts/index.yaml[the Red Hat `Helm index` file].
```
Comment (zonggen) [inline applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc:9]: To confirm, might be missing some context, but why aren't we linking users to the new `https://charts.openshift.io/index.yaml`?

File: applications/working_with_helm_charts/understanding-helm.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+== Red Hat Certification of Helm charts for OpenShift
+
+You can verify and certify your Helm charts by Red Hat for all the components you will be deploying using the Helm charts on the Red Hat {product-title}. Charts go through an automated Red Hat OpenShift certification workflow that guarantees security compliance as well as best integration and experience with the platform. Certification assures the integrity of the chart and ensures that the Helm chart works seamlessly on Red Hat OpenShift clusters.
+
+== Additional resources
+
+* For more information on how to certify your Helm charts as a Red Hat partner, see link:https://redhat-connect.gitbook.io/partner-guide-for-red-hat-openshift-and-container/helm-chart-certification/overview[Red Hat Certification of Helm charts for OpenShift].
+* For more information on OpenShift and Container certification guides for Red Hat partners, see link:https://redhat-connect.gitbook.io/partner-guide-for-red-hat-openshift-and-container/[Partner Guide for OpenShift and Container Certification].
+* For a list of the charts, see link:https://redhat-developer.github.io/redhat-helm-charts/index.yaml[the Red Hat `Helm index` file].
```
Comment (zonggen) [inline applications/working_with_helm_charts/understanding-helm.adoc:35]: Same as previous question regarding new `https://redhat-developer.github.io/redhat-helm-charts`.

File: applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc (hunk 1 above)
Comment (Preeticp) [reply applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc:9]: Hi @zonggen @Srivaralakshmi this link was given by the Helm developers to us when we first documented this. If the link needs to be changed we can do that, but can you double check this with David P and the team before we make the change?

**Review by zonggen — COMMENTED** [review]
@dperaza4dustbit Do you think we should document the differences between partners/redhat/community charts in this documentation?

File: applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc (hunk 1 above)
Comment (dperaza4dustbit) [reply applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc:9]: The new one https://charts.openshift.io/index.yaml, please

**Review by dperaza4dustbit — COMMENTED** [review]
Good catch from Allen

File: applications/working_with_helm_charts/understanding-helm.adoc [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+Helm provides the ability to:
+
+* Search through a large collection of charts stored in the chart repository.
+* Modify existing charts.
+* Create your own charts with {product-title} or Kubernetes resources.
+* Package and share your applications as charts.
+
+== Red Hat Certification of Helm charts for OpenShift
+
+You can verify and certify your Helm charts by Red Hat for all the components you will be deploying using the Helm charts on the Red Hat {product-title}. Charts go through an automated Red Hat OpenShift certification workflow that guarantees security compliance as well as best integration and experience with the platform. Certification assures the integrity of the chart and ensures that the Helm chart works seamlessly on Red Hat OpenShift clusters.
```
Comment (dperaza4dustbit) [inline applications/working_with_helm_charts/understanding-helm.adoc:29]: This does not sound well to me, I would simplify it like this: You can choose to certify your Helm Charts on Red Hat OpenShift. 
I think we can even remove that first sentence.

File: applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc (hunk 1 above)
Comment (dperaza4dustbit) [inline applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc:9]: Change the name too please from "Red Hat Helm chart repository" to "OpenShift Helm Chart repository". Also not sure it flows well. We start by saying that we can install helm charts using the cli and then jump to the developer perspective in the console. Those are two different experiences. On from the CLI and the other from the Console. Make sure they are clearly different paths.

**Review by dperaza4dustbit — COMMENTED** [review]
I think it needs so re-wording on some section I indicated.

File: applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc (hunk 1 above)
Comment (Srivaralakshmi) [reply applications/working_with_helm_charts/configuring-custom-helm-chart-repositories.adoc:9]: Changing to:

You can install Helm charts on an OpenShift Container Platform cluster using the following methods:

- Using the CLI 
- Using the **Developer** perspective of the web console.

The **Developer Catalog**, in the **Developer** perspective of the web console, displays the Helm charts available in the cluster. By default, it lists the Helm charts from the Red Hat OpenShift Helm chart repository. For a list of the charts see link:https://charts.openshift.io/index.yaml[the Red Hat `Helm index` file].

File: applications/working_with_helm_charts/understanding-helm.adoc (hunk 3 above)
Comment (Srivaralakshmi) [reply applications/working_with_helm_charts/understanding-helm.adoc:29]: @dperaza4dustbit As discussed, changing the content from _You can verify and certify_ to _You can  choose to verify and certify_.

File: applications/working_with_helm_charts/understanding-helm.adoc (hunk 2 above)
Comment (Srivaralakshmi) [reply applications/working_with_helm_charts/understanding-helm.adoc:35]: Yes, @zonggen. Thanks for the comment. Changing this to:

```suggestion
* For a list of the charts, see link:https://charts.openshift.io/index.yaml[the Red Hat Helm index file].
```

**Comment by Srivaralakshmi** [comment]
Hi @dperaza4dustbit @zonggen, Thanks for the review comments. I have addressed them all. PTAL and if there are no further comments, please provide LGTM.

**Review by zonggen — COMMENTED** [review]
LGTM!

**Review by dperaza4dustbit — APPROVED** [review]
lgtm

File: modules/odc-installing-helm-charts-using-developer-perspective.adoc [hunk 4]
```diff
@@ -1,11 +1,10 @@
-// Module included in the following assemblies:
-//
-// applications/application_life_cycle_management/odc-working-with-helm-charts-using-developer-perspective.adoc
+[id="odc-installing-helm-charts-using-developer-perspective_{context}"]
+= Installing Helm charts using the Developer perspective
 
-[id="odc-installing-helm-charts_{context}"]
-= Installing Helm charts
+You can use either the *Developer* perspective in the web console or the CLI to select and install a chart from the Helm charts listed in the *Developer Catalog*. You can create Helm releases and see them in the *Developer* perspective of the web console.
```
Comment (pmacik) [inline modules/odc-installing-helm-charts-using-developer-perspective.adoc:4]:
```suggestion
You can use either the *Developer* perspective in the web console or the CLI to select and install a chart from the Helm charts listed in the *Developer Catalog*. You can Helm releases by installing the Helm charts and see them in the *Developer* perspective of the web console.
```

In the "Understanding Helm" chapter it is stated that:
```
A running instance of the chart in a cluster is called a release. A new release is created every time a chart is installed on the cluster.
```

File: modules/helm-disabling-helm-chart-repositories.adoc [hunk 5]
```diff
@@ -1,11 +1,7 @@
...
-//
-// * cli_reference/helm/disabling-helm-chart-repositories.adoc
+[id="helm-disabling-helm-chart-repositories_{context}"]
+= Disabling Helm Chart repositories
 
-[id="helm-disabling-helm-chart-repository-in-the-cluster_{context}"]
-= Disabling Helm Chart repository in the cluster
-
-You can disable Helm Charts in the catalog by adding the `disabled` property in the `HelmChartRepository` custom resource. 
+You can disable Helm Charts in the catalog by adding the `disabled` property in the `HelmChartRepository` custom resource.
```
Comment (pmacik) [inline modules/helm-disabling-helm-chart-repositories.adoc:4]:
```suggestion
You can disable Helm Charts from a particular Helm Chart Repository in the catalog by adding the `disabled` property in the `HelmChartRepository` custom resource.
```

File: modules/helm-creating-credentials-and-certificates-to-add-helm-repositories.adoc [hunk 6]
```diff
@@ -25,7 +21,7 @@ $ oc create configmap helm-ca-cert \
 ----
 $ oc create secret generic helm-tls-configs \
 --from-file=tls.crt=/path/to/certs/client.crt \
```
Comment (pmacik) [inline modules/helm-creating-credentials-and-certificates-to-add-helm-repositories.adoc:23]:
```suggestion
$ oc create secret tls helm-tls-configs \
--cert=/path/to/certs/client.crt \
```

File: modules/helm-creating-credentials-and-certificates-to-add-helm-repositories.adoc [hunk 7]
```diff
@@ -25,7 +21,7 @@ $ oc create configmap helm-ca-cert \
...
---from-file=tls.key=/path/to/certs//client.key \
+--from-file=tls.key=/path/to/certs/client.key \
 -n openshift-config
```
Comment (pmacik) [inline modules/helm-creating-credentials-and-certificates-to-add-helm-repositories.adoc:25]:
```suggestion
--key=/path/to/certs/client.key \
-n openshift-config
```

**Comment by Srivaralakshmi** [comment]
Hi @pmacik, Thank you for your time and valuable review comments. They are helpful :)

PTAL at the revised content and provide LGTM, if there are no further comments.

**Review by Preeticp — COMMENTED** [review]
Hi @Srivaralakshmi looks great. Good work!
Please crosscheck the one comment and modify if required. Otherwise it is good to be merged. Thank you!

File: applications/working_with_helm_charts/understanding-helm.adoc [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+Helm provides the ability to:
+
+* Search through a large collection of charts stored in the chart repository.
+* Modify existing charts.
+* Create your own charts with {product-title} or Kubernetes resources.
+* Package and share your applications as charts.
+
+== Red Hat Certification of Helm charts for OpenShift
+
+You can choose to verify and certify your Helm charts by Red Hat for all the components you will be deploying using the Helm charts on the Red Hat {product-title}. Charts go through an automated Red Hat OpenShift certification workflow that guarantees security compliance as well as best integration and experience with the platform. Certification assures the integrity of the chart and ensures that the Helm chart works seamlessly on Red Hat OpenShift clusters.
```
[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply applications/working_with_helm_charts/understanding-helm.adoc]: I have addressed the comment. Thank you!

**Pull request merged**